### PR TITLE
PuTTY Tutorial

### PR DESCRIPTION
**Pull request by jay-hankins**
The basic PuTTY tutorial looks bangin'

### CODE REVIEW

File: tutorials/shells/cygwin.md [hunk 1]
```diff
@@ -1,50 +1,61 @@
...
 Copyright 2015 [Jay Hankins](http://jayhankins.me)
 
-## What is it?
-From the [Cygwin](http://cygwin.org) homepage, Cygwin is:
-> * a large collection of GNU and Open Source tools which provide functionality similar to a Linux distribution on Windows.
-> * a DLL (cygwin1.dll) which proides substantial POSIX API functionality.
+What is it?
+-----------
+
+From the [Cygwin](http://cygwin.org) homepage, Cygwin is: > * a large collection of GNU and Open Source tools which provide functionality similar to a Linux distribution on Windows. > * a DLL (cygwin1.dll) which proides substantial POSIX API functionality.
```
Comment (scottopell) [inline tutorials/shells/cygwin.md]: this markdown is broken

File: tutorials/shells/cygwin.md [hunk 2]
```diff
@@ -1,50 +1,61 @@
...
 
-6. Mark a package to be downloaded by clicking the `⟳` to the left of "Skip".
-![Marked package to install](https://github.com/jay-hankins/CSToolsCourse/raw/master/tutorials/shells/cygwin_screenshots/search_openssh.PNG)
+7.	Click "Next" when you've selected your desired packages, and "Next" again to download and install them. Wait patiently while the Cygwin installer downloads and installs the packages and their dependencies.
 
+8.	Click "Finish" to complete the installation.
 
-7. Click "Next" when you've selected your desired packages, and "Next" again to download and install them. Wait patiently while the Cygwin installer downloads and installs the packages and their dependencies.
-8. Click "Finish" to complete the installation.
+How do I use it?
```
Comment (scottopell) [inline tutorials/shells/cygwin.md]: Why the change on all of these headers from 
```
## header 2
```
to
```
header 2
------------
```

Comment (jay-hankins) [inline tutorials/shells/cygwin.md]: I set up my editor to do it this way because that's my preference but I can change it back.

Comment (scottopell) [inline tutorials/shells/cygwin.md]: Got it. Idk, if we want to have a certain style across all files. Just something I noticed. @leviathan747 may have thoughts

**Comment by leviathan747** [comment]
Looks good. If you would, add a comment about using PuTTY on serial ports (no need to go in depth). Useful for any project using hardware. Like connecting to a Raspberry Pi on UART.

**Comment by jay-hankins** [comment]
Added serial instructions.

**Comment by scottopell** [comment]
minor detail, and I may be missing something, but in

https://github.com/jay-hankins/CSToolsCourse/blob/master/tutorials/shells/cygwin.md#how-do-i-get-it 
step 6, I can't figure out where the ⟳ left of skip is. Or where skip is for that matter.

Maybe a circle or something on the screenshot would be helpful?

**Comment by jay-hankins** [comment]
@scottopell Good point. I just looked back over it and I'm not really sure what I was trying to say. My guess is there was a state change before I took the screenshot so whatever I was referencing isn't in view anymore. I'll tweak that.

**Comment by jay-hankins** [comment]
@leviathan747 or @scottopell I guess this never got merged in, but it looks ready to me.
